packages/core/typescript/TypeScriptFileUpdate.ts

```diff
@@ -36,6 +36,12 @@ export class TypeScriptFileUpdate {
 		this.initState();
 	}
 
+	public transform(source: ts.SourceFile | ts.SourceFile[],
```

Comment:
What's the point of this when all it's doing is acting as a proxy of the same method?
And also causes the usage code to be forced to type assert as well, since this isn't typed quite like the original TS method. IF we have to keep this, do also copy the generic typing from the wrapped method and drop all the extra type asserts you were forced to add

Reply:
This is a proxy that is currently used for the tests. It will be gone with https://github.com/IgniteUI/igniteui-cli/pull/1238